thunder/executors/passes.py

```diff
@@ -129,8 +129,16 @@ def visit_(bsym: BoundSymbol) -> transforms.VISIT_TYPE:
 
 
 def transform_for_execution(trace: TraceCtx, executors_list: Sequence[Executor]) -> TraceCtx:
+    import torch
+
```

Comment:
It doesn't seem that this import is needed here.

Reply:
really?

Reply:
My bad I see it's used for `torch.distributed.is_available`.